packages/@lwc/engine-core/src/framework/__tests__/CustomElementConstructor-getter.spec.ts

```diff
@@ -1,16 +0,0 @@
-/*
```

Comment:
It is the only unit test in the `@lwc/engine-core` package. As the `LigtningElement.CustomElement` constructor API has to be overridden in `@lwc/engine-dom` and `@lwc/engine-server`, I don't see the point of keeping this orphan test.